test/db/rzil/hexagon

```diff
@@ -0,0 +1,389 @@
+EXPECT=<<EOF
+EOF
+RUN
+
+NAME=Run test_reorder
+FILE=bins/elf/hexagon/rzil/test_reorder
+TIMEOUT=5
+CMDS=<<EOF
+aezi
+aezsu loc.pass
+EOF
+EXPECT=<<EOF
+EOF
+RUN
+
+NAME=Run test_packet
+FILE=bins/elf/hexagon/rzil/test_packet
+TIMEOUT=5
+CMDS=<<EOF
+aezi
+aezsu loc.pass
+EOF
+EXPECT=<<EOF
+EOF
+RUN
+
+NAME=Run test_mpyi
+FILE=bins/elf/hexagon/rzil/test_mpyi
+TIMEOUT=5
+CMDS=<<EOF
+aezi
+aezsu loc.pass
+EOF
+EXPECT=<<EOF
+EOF
+RUN
+
+NAME=Run test_lsr
+FILE=bins/elf/hexagon/rzil/test_lsr
+TIMEOUT=5
+CMDS=<<EOF
+aezi
+aezsu loc.pass
+EOF
+EXPECT=<<EOF
+EOF
+RUN
+
+NAME=Run test_jmp
+FILE=bins/elf/hexagon/rzil/test_jmp
+TIMEOUT=5
+CMDS=<<EOF
+aezi
+aezsu loc.pass
+EOF
+EXPECT=<<EOF
+EOF
+RUN
+
+NAME=Run test_hwloops
+FILE=bins/elf/hexagon/rzil/test_hwloops
+TIMEOUT=5
+CMDS=<<EOF
+aezi
+aezsu loc.pass
+EOF
+EXPECT=<<EOF
+EOF
+RUN
+
+NAME=Run test_hl
+FILE=bins/elf/hexagon/rzil/test_hl
+TIMEOUT=5
+CMDS=<<EOF
+aezi
+aezsu loc.pass
+EOF
+EXPECT=<<EOF
+EOF
+RUN
+
+NAME=Run test_fibonacci
+FILE=bins/elf/hexagon/rzil/test_fibonacci
+TIMEOUT=5
+CMDS=<<EOF
+aezi
+aezsu loc.pass
+EOF
+EXPECT=<<EOF
+EOF
+RUN
+
+NAME=Run test_ext
+FILE=bins/elf/hexagon/rzil/test_ext
+TIMEOUT=5
+CMDS=<<EOF
+aezi
+aezsu loc.pass
+EOF
+EXPECT=<<EOF
+EOF
+RUN
+
+NAME=Run test_dotnew
+FILE=bins/elf/hexagon/rzil/test_dotnew
+TIMEOUT=5
+CMDS=<<EOF
+e io.cache=1
+aezi
+aezsu loc.pass
+EOF
+EXPECT=<<EOF
+EOF
+RUN
+
+NAME=Run test_cmp
+FILE=bins/elf/hexagon/rzil/test_cmp
+TIMEOUT=5
+CMDS=<<EOF
+aezi
+aezsu loc.pass
+EOF
+EXPECT=<<EOF
+EOF
+RUN
+
+NAME=Run test_clobber
+FILE=bins/elf/hexagon/rzil/test_clobber
+TIMEOUT=5
+CMDS=<<EOF
+e io.cache=1
+aezi
+aezsu loc.pass
+EOF
+EXPECT=<<EOF
+EOF
+RUN
+
+NAME=Run test_call
+FILE=bins/elf/hexagon/rzil/test_call
+TIMEOUT=5
+CMDS=<<EOF
+aezi
+aezsu loc.pass
+EOF
+EXPECT=<<EOF
+EOF
+RUN
+
+NAME=Run test_bitsplit
+FILE=bins/elf/hexagon/rzil/test_bitsplit
+TIMEOUT=5
+CMDS=<<EOF
+aezi
+aezsu loc.pass
+EOF
+EXPECT=<<EOF
+EOF
+RUN
+
+NAME=Run test_bitcnt
+FILE=bins/elf/hexagon/rzil/test_bitcnt
+TIMEOUT=5
+CMDS=<<EOF
+aezi
+aezsu loc.pass
+EOF
+EXPECT=<<EOF
+EOF
+RUN
+
+NAME=Run test_abs
+FILE=bins/elf/hexagon/rzil/test_abs
+TIMEOUT=5
+CMDS=<<EOF
+aezi
+aezsu loc.pass
+EOF
+EXPECT=<<EOF
+EOF
+RUN
+
+NAME=Run first
+FILE=bins/elf/hexagon/rzil/first
+TIMEOUT=5
+CMDS=<<EOF
+aezi
+aezsu loc.pass
+EOF
+EXPECT=<<EOF
+EOF
+RUN
+
+
+NAME=Run v68_scalar
+TIMEOUT=240
+FILE=bins/elf/hexagon/rzil/v68_scalar
+CMDS=<<EOF
+e io.cache=1
+aezi
+aezsu 0x000210b0 # End of main
+EOF
+EXPECT=<<EOF
+EOF
+RUN
+
+NAME=Run v73_scalar
+TIMEOUT=240
+FILE=bins/elf/hexagon/rzil/v73_scalar
+CMDS=<<EOF
+e io.cache=1
+aezi
+aezsu 0x00020e40 End of main
+EOF
+EXPECT=<<EOF
+EOF
+RUN
+
+NAME=Run test-vma
+TIMEOUT=240
+FILE=bins/elf/hexagon/rzil/test-vma
+CMDS=<<EOF
+e io.cache=1
+aezi
+aezsu 0x00020214 # End of main
+EOF
+EXPECT=<<EOF
+EOF
+RUN
+
+# NAME=Run load_align
+# FILE=bins/elf/hexagon/rzil/load_align
+# CMDS=<<EOF
+# aezi
+# aezsu loc.pass
+# EOF
+# EXPECT=<<EOF
+# EOF
+# RUN
```

Comment:
?

Reply:
It executes until it hits the pass location. If it fails the test will time out since it never reaches it. Basically the poor mans attempt to check if everything was fine. Since we cannot reasonably run rz-tracetest.

Reply:
yes but this is commented. (thus not executed)

Reply:
Ouh, yes. Just tested it. Something is wrong there and I forgot about it.